ws.py

```diff
@@ -280,6 +280,207 @@ def build_regions():
+            "PRT",
+            "ROU",
+            "SVK",
+            "SVN",
+            "SWE",
+        ],
+        "African Group": [
+            "AGO",
+            "BDI",
+            "BEN",
+            "BFA",
+            "BWA",
+            "CAF",
+            "CIV",
+            "CMR",
+            "COD",
+            "COG",
+            "COM",
+            "CPV",
+            "DJI",
+            "DZA",
+            "EGY",
+            "ERI",
+            "ETH",
+            "GAB",
+            "GHA",
+            "GIN",
+            "GMB",
+            "GNB",
+            "GNQ",
+            "KEN",
+            "LBR",
+            "LBY",
+            "LSO",
+            "MAR",
+            "MDG",
+            "MLI",
+            "MOZ",
+            "MRT",
+            "MUS",
+            "MWI",
+            "NAM",
+            "NER",
+            "NGA",
+            "RWA",
+            "SDN",
+            "SEN",
+            "SLE",
+            "SOM",
+            "SSD",
+            "STP",
+            "SWZ",
+            "SYC",
+            "TCD",
+            "TGO",
+            "TUN",
+            "TZA",
+            "UGA",
+            "ZAF",
+            "ZMB",
+            "ZWE",
+        ],
+        "Northern America": ["CAN", "USA"],
+        "Australasia": ["AUS", "NZL"],
+        "Umbrella": ["AUS", "CAN", "ISL", "ISR", "JPN", "KAZ", "NOR", "NZL", "UKR", "USA"],
+        "G7": ["CAN", "DEU", "FRA", "GBR", "ITA", "JPN", "USA"],
+        "G20": ["ARG", "AUS", "BRA", "CAN", "CHN", "DEU", "FRA", "GBR", "IND", "IDN", "ITA", "JPN", "KOR", "MEX", "RUS", "SAU", "TUR", "USA", "ZAF"],
+        "FSM": ["MHL", "PLW", "NRU"],
+        "LDC": [
+            "AFG",
+            "AGO",
+            "BDI",
+            "BEN",
+            "BFA",
+            "BGD",
+            "BOL",
+            "BTN",
+            "CAF",
+            "CIV",
+            "CMR",
+            "COD",
+            "COM",
+            "CPV",
+            "DJI",
+            "DZA",
+            "EGY",
+            "ERI",
+            "ETH",
+            "GHA",
+            "GIN",
+            "GMB",
+            "GNB",
+            "GNQ",
+            "HTI",
+            "KEN",
+            "KHM",
+            "KIR",
+            "LAO",
+            "LBR",
+            "LSO",
+            "MDG",
+            "MLI",
+            "MMR",
+            "MRT",
+            "MUS",
+            "MWI",
+            "NER",
+            "NGA",
+            "NIC",
+            "NPL",
+            "PAK",
+            "PHL",
+            "PNG",
+            "PRY",
+            "RWA",
+            "SDN",
+            "SEN",
+            "SLE",
+            "SLB",
+            "SOM",
+            "SSD",
+            "STP",
+            "SWZ",
+            "SYC",
+            "TCD",
+            "TGO",
+            "TLS",
+            "TZA",
+            "UGA",
+            "VUT",
+            "YEM",
+            "ZMB",
+            "ZWE",
+        ],
+        "SIDS": [
```

Comment:
SIDS are 29 countries in this list but 39 in the excel sheet. E.g TLS (Timor-Leste) is missing in this list.